scripts/tests/bootstrapper/lotus.dockerfile

```diff
@@ -0,0 +1,55 @@
+# Lotus binaries image, to be used in the local devnet with Forest.
+FROM golang:1.21-bullseye AS lotus-builder
+
+RUN apt-get update && apt-get install -y curl ca-certificates build-essential clang ocl-icd-opencl-dev ocl-icd-libopencl1 jq libhwloc-dev 
+
+WORKDIR /lotus
+
+# Install rust toolchain for rebuilding `filecoin-ffi`
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --no-modify-path --profile minimal
+
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+ARG LOTUS_VERSION
+RUN git clone --depth 1 --branch ${LOTUS_VERSION} https://github.com/filecoin-project/lotus.git .
+
+# Replace the default bootstrap peers with a pre-defined one. This is needed on this level because the bootstrap peers are compiled into the binary and not configurable at runtime.
+ARG BOOTSTRAPPER
+RUN echo ${BOOTSTRAPPER} > build/bootstrap/calibnet.pi
+
+# Apply a patch to Lotus so that local IPs are not discarded
+COPY patch-lotus.diff .
+RUN git apply patch-lotus.diff
+
+# https://github.com/Filecoin-project/filecoin-ffi?tab=readme-ov-file#building-from-source
+RUN CGO_CFLAGS_ALLOW="-D__BLST_PORTABLE__" \
+    CGO_CFLAGS="-D__BLST_PORTABLE__" \
+    FFI_USE_BLST_PORTABLE="1" \
+    FFI_USE_GPU="0" \
+    make calibnet && strip lotus*
+
+FROM ubuntu:22.04
+
+# Need to copy the relevant shared libraries from the builder image.
+# See https://github.com/filecoin-project/lotus/blob/master/Dockerfile
```

Comment:
I'd prefer a permalink to a tagged release

Reply:
It's not useful in the context of a constantly updated tag. Do you agree that bumping the Lotus version only in the environment file is more maintainable that going through all the linked permalinks?